src/utils/polygon.h

```diff
@@ -451,6 +451,13 @@ class PolygonRef : public ConstPolygonRef
      * 
      * Removes verts which are connected to line segments which are both too small.
      * Removes verts which detour from a direct line from the previous and next vert by a too small amount.
+     *
+     * Criteria:
+     * 1. Never remove a vertex if either of the connceted segments is larger than \p smallest_line_segment
+     * 2. Never remove a vertex if the distance between that vertex and the final resulting polygon would be higher than \p allowed_error_distance
+     * 3. Simplify uses a heuristic and doesn't neccesarily remove all removable vertices under the above criteria.
+     * 4. But simplify may never violate these criteria.
+     * 5. Unless the segments or the distance is smaller than the rounding error of 5 micron
```

Comment:
This confuses me. Does it refer to 4? (if so, it should be a part of 4) If not, I don't understand what it does refer to.

Reply:
Yes it refers to 4.